tendrl/commons/objects/__init__.py

```diff
@@ -201,7 +202,9 @@ def load(self):
                 pass
 
         _copy = self._copy_vars()
-
+        # Check if self.value already set, use it
+        if self.value.find('{') < 0:
```

Comment:
self.render is already called on L167. It safe to assume self.value is rendered correctly, no need to check